simulation/tests/edge_observation/test_digital_io.py

```
@@ -0,0 +1,361 @@
+
+
+def test_a_snapshot_cannot_repeat_one_input_name():
+    with pytest.raises(EdgeAdapterError):
+        _snapshot(
+            _input(SENSOR_STATION_OPEN, "equipment_ready", True),
+            _input(SENSOR_ZONE_OPEN, "equipment_ready", True),
+        )
+
+
+def test_a_mismatched_device_identity_is_rejected(kit):
+    result, by_channel = convert_one(
```

Ruff (0.16.1)

[warning] Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)